characters/cryo/kaeya.md

```diff
@@ -85,6 +85,7 @@ Coalescing the frost in the air, Kaeya summons 3 icicles that revolve around him
 * **Glacial Waltz** does not have any hitlag.
 * **Glacial Waltz** hits around 13 times while standing stationary.
 * The cooldown of **Glacial Waltz** starts on frame 49. The duration also starts counting from this frame.
+* Each Icicle has a ~1 second ICD.
```

Comment:
Does Kaeya's Burst ICD not just follow the conventional ICD for elemental application? Or is this the jank lockout where you can't damage enemies with his burst?

Also, I don't know Git etiquette so I'm just gonna hit the shiny green button and hope that's the right one.

Reply:
The ICD is the jank lockout, I didn't know how else to describe it besides an ICD.

You're correct on the git part.